lib/logstash/filters/mutate.rb

```diff
@@ -268,7 +273,11 @@ def convert(event)
       else
         value = converter.call(original)
       end
-      event[field] = value
+      if value.nil?
```

Comment:
Perhaps not.  Is it a completely safe assumption that none of the other convert methods will ever send a nil value?